apps/sms/test/marionette/conversation_test.js

```diff
@@ -491,4 +503,101 @@ marionette('Conversation Panel Tests', function() {
       });
     });
   });
+
+  suite('Add recipients from thread', function() {
+    var conversationView;
+    setup(function() {
+      var thread = ThreadGenerator.generate({
+        numberOfMessages: 1
+      });
+      storage.setMessagesStorage([thread], ThreadGenerator.uniqueMessageId);
```

Comment:
More or less ;) When I think there could be an intermittent, I respin the right Gij (usually Gij16 or 17, but we need to check the logs to find the right one)